### PR TITLE
Feature/score aggregation

### PR DESCRIPTION
**Pull request by DaQuirm**

### CODE REVIEW

File: app/game-session.coffee [hunk 1]
```diff
@@ -18,14 +19,30 @@ class GameSession
 		@participants = new nx.Collection
 		@participants_by_id = new Map
 
+		@scorekeeper = new Scorekeeper
+			default_score: GameSession.ROUND_DURATION
```
Comment (kirilknysh) [inline app/game-session.coffee]: All rounds gonna have the same duration?

Comment (DaQuirm) [inline app/game-session.coffee]: Hmm, should also be configurable, you think?
How about we create another "future"/"discussion" issue for that and keep the time constant for now?

Comment (kirilknysh) [inline app/game-session.coffee]: sure!

**Comment by kirilknysh** [comment]
I would propose to merge it now. I'll be working on it next couple of hours and will let know if smth is wrong.

**Comment by DaQuirm** [comment]
OK let's merge it :shipit:

**Pull request merged**